src/main/java/com/tutorhub/model/course/Content.java

```diff
@@ -0,0 +1,41 @@
+package com.tutorhub.model.course;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.UUID;
+
+@Entity
+@Table(name = "content")
+@Getter
+@Setter
+public class Content {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  @Column(name = "fileId")
```

Comment:
it is preferred to use `snake_case` in SQL, and Hibernate will use generated snake_case names automatically, so you don't need to write `@Column(name = "title")`

Reply:
Fixed